sdk/keyvault/Azure.ResourceManager.KeyVault/README.md

```diff
@@ -1,69 +1,51 @@
+## Need help?
 
-## Examples
-
-Include code snippets and short descriptions for each task you listed in the [Introduction](#introduction) (the bulleted list). Briefly explain each operation, but include enough clarity to explain complex or otherwise tricky operations.
-
-## Troubleshooting
-
-Describe common errors and exceptions, how to "unpack" them if necessary, and include guidance for graceful handling and recovery.
-
-Provide information to help developers avoid throttling or other service-enforced errors they might encounter. For example, provide guidance and examples for using retry or connection policies in the API.
-
-If the package or a related package supports it, include tips for logging or enabling instrumentation to help them debug their code.
-
-## Next steps
-
-* Provide a link to additional code examples, ideally to those sitting alongside the README in the package's `/samples` directory.
-* If appropriate, point users to other packages that might be useful.
-* If you think there's a good chance that developers might stumble across your package in error (because they're searching for specific functionality and mistakenly think the package provides that functionality), point them to the packages they might be looking for.
+-   File an issue via [Github
+    Issues](https://github.com/Azure/azure-sdk-for-net/issues)
+-   Check [previous
+    questions](https://stackoverflow.com/questions/tagged/azure+.net)
+    or ask new ones on StackOverflow using azure and .NET tags.
 
 ## Contributing
 
-This is a template, but your SDK readme should include details on how to contribute code to the repo/package.
+For details on contributing to this repository, see the contributing
+guide.
+
+This project welcomes contributions and suggestions. Most contributions
+require you to agree to a Contributor License Agreement (CLA) declaring
+that you have the right to, and actually do, grant us the rights to use
+your contribution. For details, visit <https://cla.microsoft.com>.
 
-<!-- LINKS -->
-[style-guide-msft]: https://docs.microsoft.com/style-guide/capitalization
-[style-guide-cloud]: https://worldready.cloudapp.net/Styleguide/Read?id=2696&topicid=25357
+When you submit a pull request, a CLA-bot will automatically determine
+whether you need to provide a CLA and decorate the PR appropriately
+(e.g., label, comment). Simply follow the instructions provided by the
+bot. You will only need to do this once across all repositories using
+our CLA.
 
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Ftemplate%2FAzure.Template%2FREADME.png)
+This project has adopted the Microsoft Open Source Code of Conduct. For
+more information see the Code of Conduct FAQ or contact
+<opencode@microsoft.com> with any additional questions or comments.
```

Comment:
All these READMEs should probably have the impressions image we use for stats. See data-plane READMEs for examples.

Reply:
not all management RPs currently have data-plane counterpart, is there an impression image generation tool ?